system/jlib/jstats.cpp

```diff
@@ -1919,6 +1919,10 @@ class CStatisticCollection : public CInterfaceOf<IStatisticCollection>
 //other public interface functions
     void addStatistic(StatisticKind kind, unsigned __int64 value)
     {
+#if 1 //def _DEBUG
```

Comment:
This will be debug only in the final PR - pushed to run the full regression suite.

Reply:
It would be useful to have a comment to say that addStatistic shouldn't ever add the same kind multiple times.